backend/tests/slack/events/fuzz_tests/user_joind_channel/gsoc_test.py

```
@@ -0,0 +1,41 @@
+    )
+    def test_check_gsoc_handler(self, channel_id):
+        gsoc_module = __import__(
+            "apps.slack.events.member_joined_channel.gsoc",
+            fromlist=["gsoc_handler"],
+        )
+        check_gsoc_handler = getattr(
+            gsoc_module,
+            "check_gsoc_handler",
+            lambda x: x.get("channel") == OWASP_GSOC_CHANNEL_ID,
+        )
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)